modules/services/xcape.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.xcape;
+
+in
+
+{
+  meta.maintainers = [ maintainers.nickhu ];
+
+  options = {
+    services.xcape = {
+      enable = mkEnableOption "xcape";
+
+      package = mkOption {
```

Comment:
Since Nixpkgs only contains a single xcape package I think this this option can be removed. If a user wants to provide a different package they can use an overlay.